packages/@aws-cdk/aws-glue-alpha/lib/jobs/pyspark-etl-job.ts

```diff
@@ -84,16 +79,12 @@ export class PySparkEtlJob extends Job {
 
   /**
    * PySparkEtlJob constructor
-   *
-   * @param scope
-   * @param id
-   * @param props
```

Comment:
i have no problem with these doctags, if they were filled out. i'm also happy to omit them entirely, so just deleting here